source/content/guides/composer-convert.md

```diff
@@ -201,20 +250,23 @@ git commit -m "Convert to Composer based install"
 You should see a large amount of files committed to the new branch we created earlier.
 
 ## Deploy
-You've now committed the code to a branch. If your site has multidev, you can deploy that branch directly to a new multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composify -- wd-show` to inspect the watchdog logs, or follow the directions on [our documentation on log collection](/logs).
 
-Once you have confirmed the site is working, merge `composify` into `master`, and follow the standard workflow to QA a code change before going live.
+You've now committed the code to a branch. If your site has Multidev, you can deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composerify -- wd-show` to inspect the watchdog logs, or follow the directions on our documentation on [log collection](/logs).
+
+Once you have confirmed the site is working, merge `composerify` into `master`, and follow the standard workflow to QA a code change before going live.
 
-If your plan does not include multidev, you will have to merge to master before deploying, then follow the rest of the steps above. If you have a local development solution, consider testing your `composify` branch locally before merging.
+If your plan does not include Multidev, you will have to merge to master before deploying, then follow the rest of the steps above. If you have a local development solution, consider testing your `composerify` branch locally before merging.
 
 ## Change Upstreams
+
 Your Pantheon site is no longer compatible with traditional upstream updates. Avoid confusion by moving your site to an empty upstream:
 
 ```bash{promptUser:user}
 terminus site:upstream:set $site empty
 ```
 
 ## Ongoing Core Updates
```

Comment:
Heads up to update this part to recommend use of the dashboard's check for updates / apply updates functionality when using Integrated Composer